sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample7_ManageCustomModels.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Training;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    [LiveOnly]
+    public partial class FormRecognizerSamples
+    {
+        [Test]
+        public async Task ManageCustomModels()
+        {
+            string endpoint = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_ENDPOINT");
+            string apiKey = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_API_KEY");
+            string trainingFileUrl = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_BLOB_CONTAINER_SAS_URL");
+
+            #region Snippet:FormRecognizerSample7ManageCustomModels
+
+            FormTrainingClient client = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Check number of models in the FormRecognizer account, and the maximum number of models that can be stored.
+            AccountProperties accountProperties = client.GetAccountProperties();
+            Console.WriteLine($"Account has {accountProperties.CustomModelCount} models.");
+            Console.WriteLine($"It can have at most {accountProperties.CustomModelLimit} models.");
+
+            // List the first ten or fewer models currently stored in the account.
+            Pageable<CustomFormModelInfo> models = client.GetModelInfos();
+
+            foreach (CustomFormModelInfo modelInfo in models.Take(10))
+            {
+                Console.WriteLine($"Custom Model Info:");
+                Console.WriteLine($"    Model Id: {modelInfo.ModelId}");
+                Console.WriteLine($"    Model Status: {modelInfo.Status}");
+                Console.WriteLine($"    Created On: {modelInfo.CreatedOn}");
+                Console.WriteLine($"    Last Modified: {modelInfo.LastModified}");
+            }
+
+            // Create a new model to store in the account
+            CustomFormModel model = await client.StartTrainingAsync(new Uri(trainingFileUrl)).WaitForCompletionAsync();
```

Comment:
Not sure we need to show this here, as we've done it in other samples.
for `GetCustomModel` we could just use the first modelId returned from `GetModelInfos`

Reply:
I wanted to be able to delete a model created by this sample, and not a different model where I don't know the source.  What do you think?

Reply:
hmm good point. In our case it won't matter, but if a customer runs this, then yes. 
Thanks for pointing that out